jest-assessment-week-2-borja/code-challenges.test.js

```diff
@@ -0,0 +1,315 @@
+// Use conditional statements if/else it to return divisible ot not divisible
+// input: object {numberKey: numberValue}
+// output: `${object.number} is divisible by three.` || `${object.number} is not divisible by three.`
+// yarn jest should Pass (green)
+
+const divisibleByThree = (object) => {
+    if (object.number % 3 === 0 ) {
+        return `${object.number} is divisible by three.`
+    } else if (object.number % 3 !== 0) {
+        return `${object.number} is not divisible by three.`
+    }
+}
+
+// PASS  Desktop/week-2-assessment-borja-x13/jest-assessment-week-2-borja/code-challenges.test.js
+
+
+
+// --------------------2) Create a function that takes in an array of words and returns an array with all the words capitalized.
+
+// a) Create a test with expect statements for each of the variables provided.
+
+const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
+// Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
+const randomNouns2 = ["temperature", "database", "chopsticks", "mango"]
+// Expected output: ["Temperature", "Database", "Chopsticks", "Mango"]
+
+//------------------------------------ My Test Code ------------------------------------
+
+// pseudo code:
+//I will use the jest test template to test a function "capitalized"
+// it "takes an array "words" and returns an array with the first letter of each word capitalized"
+// expect(capitalized(["streetlamp", "potato", "teeth", "conclusion", "nephew"])).toEqual(["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"])
+// expect(capitalized(["temperature", "database", "chopsticks", "mango"])).toEqual(["Temperature", "Database", "Chopsticks", "Mango"])
+// input: since there isn't a function to test, there is no input and I expect a fail (red)
+// output: should output a ReferenceError: capitalized is not defined
+
+describe ("capitalized", () => {
+    it ("takes an array of words and returns an array with the first letter of each word capitalized", () => {
+        expect(capitalized(["streetlamp", "potato", "teeth", "conclusion", "nephew"])).toEqual(["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"])
+        expect(capitalized(["temperature", "database", "chopsticks", "mango"])).toEqual(["Temperature", "Database", "Chopsticks", "Mango"])
+    })
+})
+
+// FAIL  Desktop/week-2-assessment-borja-x13/jest-assessment-week-2-borja/code-challenges.test.js
+//   ● capitalized › takes an array of words and returns an array with the first letter of each word capitalized
+//     ReferenceError: capitalized is not defined
+
+
+// b) Create the function that makes the test pass.
+
+// pseudo: 
+// I will declare a function capitalized that accepts a parameter of an array of words
+// Function will use .map() function to iterate through the array and change each string within to uppercase the first letter
+// Use conditional statements if/else it to return divisible ot not divisible
```

Comment:
How will you target that first letter? You could add that to your pseudocode and I don't think we are using conditional statement for this one.